pkg/operator/config_reloader.go

```diff
@@ -230,6 +260,30 @@ func CreateConfigReloader(name string, options ...ReloaderOption) v1.Container {
 
 	boolFalse := false
 	boolTrue := true
+	if !configReloader.runOnce && configReloader.config.IsProbes {
```

Comment:
may be worth a unit test to assert that probes are added if the config is set.

Reply:
add unittest case